tests/Models/DAIInterestRateModelTest.js

```diff
@@ -68,19 +68,20 @@ async function getKovanFork() {
   return {kovan, root, accounts};
 }
 
-describe('DAIInterestRateModelV2', () => {
+describe('DAIInterestRateModelV3', () => {
   describe("constructor", () => {
     it.skip("sets jug and ilk address and pokes", async () => {
```

Comment:
The fork could use the same style that's in the sims now, e.g. `https://mainnet-eth.compound.finance/@10331520` - we have redirects with infura keys for mainnet and testnets now so we can run this with a key that has archive access

Reply:
However the fork tests also slow down test suites significantly, so skipping also might be better once its working